test/run-tests.py

```diff
@@ -702,6 +705,9 @@ def main(args):
   parser.add_argument('--no-roundtrip',
                       help='don\'t run roundtrip.py on all tests',
                       action='store_false', default=True, dest='roundtrip')
+  parser.add_argument('-p', '--print-cmd',
```

Comment:
I didn't consider that but it would have made this change more invasive, since not all sub-commands support -p I think.